test/unit/error.test.ts

```diff
@@ -546,6 +548,56 @@ describe('MongoErrors', () => {
       expect(isResumableError(new MongoNetworkError('ah!'))).to.be.true;
     });
 
+    it('should return false for Error with code', () => {
+      const errorWithCode = new (class extends Error {
+        code = MONGODB_ERROR_CODES.CursorNotFound;
+      })();
+      expect(isResumableError(errorWithCode)).to.be.false;
+    });
+
+    it('should return false nullish arguments', () => {
+      expect(isResumableError()).to.be.false;
+      expect(isResumableError(null)).to.be.false;
+      expect(isResumableError(null, null)).to.be.false;
+    });
+
+    it('should return true for labelless MongoError with CursorNotFound code', () => {
+      const mongoError = new MongoError('ah!');
+      mongoError.code = MONGODB_ERROR_CODES.CursorNotFound;
+      expect(isResumableError(mongoError)).to.be.true;
+      expect(isResumableError(mongoError, 9)).to.be.true;
+      expect(isResumableError(mongoError, 8)).to.be.true;
+    });
+
+    it('should return true for labeled MongoError above wireVersion 9', () => {
```

Comment:
if you don't use a separate it block for labeled errors that don't have a wire version above 9, then you should be explicit about the test case wording as well as group the false assertions together separately from the true assertions

```suggestion
    it('should return true for labeled MongoError only if the wireVersion is at least 9', () => {
```